FirstApp/FirstApp.tex

```diff
@@ -853,7 +853,7 @@ \subsection{\ind{SqueakSource}: a sharing platform for Monticello}
 \needlines{5}
 \begin{code}{}
 MCHttpRepository
-    location: 'http://www.squeaksource.com/!\emph{YourProject}!'

```

Comment:
Is this a correct change? It reads like this was intended to explain how to create ones own squeaksource repo. (Same for the next change)

Reply:
You're right. I have misread that section.